react/features/screenshot-capture/ScreenshotCaptureSummary.js

```diff
@@ -54,6 +55,31 @@ export default class ScreenshotCaptureSummary {
         this._initScreenshotCapture = this._initScreenshotCapture.bind(this);
         this._streamWorker = new Worker(timerWorkerScript, { name: 'Screenshot capture worker' });
         this._streamWorker.onmessage = this._handleWorkerAction;
+
+        this._initializedRegion = false;
+    }
+
+    /**
+     * Make a call to backend for region selection.
+     *
+     * @returns {void}
+     */
+    async _initRegionSelection() {
+        const { _screenshotHistoryRegionUrl } = this._state['features/base/config'];
+        const conference = getCurrentConference(this._state);
+        const sessionId = conference.getMeetingUniqueId();
+        const { jwt } = this._state['features/base/jwt'];
+
+        const headers = {
+            ...jwt && { 'Authorization': `Bearer ${jwt}` }
+        };
+
+        await fetch(`${_screenshotHistoryRegionUrl}/${sessionId}`, {
```

Comment:
maybe we should guard for this url to be truthy, I see it's always the case at this moment, but things might change